src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1234,48 +1236,68 @@ public static string GetDecoratedSymbolName(SymbolReference symbolReference)
             await requestContext.SendResult(result);
         }
 
+        // Since the NamedPipeConnectionInfo type is only available in 5.1+
+        // we have to use Activator to support older version of PS.
+        // This code only lives in the v1.X of the extension.
+        // The 2.x version of the code can be found here:
+        // https://github.com/PowerShell/PowerShellEditorServices/pull/881
+        private static Type _namedPipeConnectionInfoType = Type.GetType("System.Management.Automation.Runspaces.NamedPipeConnectionInfo, System.Management.Automation");
+        private static ConstructorInfo _namedPipeConnectionInfoCtor = _namedPipeConnectionInfoType.GetConstructor(new [] { typeof(int) });
```

Comment:
- Can be `private static readonly`

- Prefix with `s_` (assuming we're still shooting for [corefx's style rules](https://github.com/dotnet/corefx/blob/master/Documentation/coding-guidelines/coding-style.md))

- Group with other fields at the top of the file

Reply:
fixed all